internal/clientconn/listener.go

```diff
@@ -91,6 +104,17 @@ func (l *Listener) Run(ctx context.Context) error {
 		logger.Sugar().Infof("Listening on %s ...", l.Unix())
 	}
 
+	if l.Listener.TLS != "" {
+		var err error
+		if l.tlsListener, err = setupTLSListener(l.Listener.TLS, l.Listener.TLSCertFile, l.Listener.TLSKeyFile); err != nil {
+			return lazyerrors.Error(err)
+		}
```

Comment:
codecov says this is not covered by tests.  Should it?

Reply:
We don't have a test for that. I tried to write one in `embedded_test.go`. It's not easy to do so at the moment.